src/xdocs/property_types.xml

```diff
@@ -673,7 +673,7 @@
     <section name="Pattern[]">
       <p>
         This type represents a set of patterns. The string representation
-        is parsed as a set of forward slash ('/') separated patterns.
+        is parsed as a set of comma (',') separated patterns.
```

Comment:
Add this text that can be seen on other properties.

> Alternatively, a property of this type can be supplied multiple times which is equivalent to a set of comma separated patterns

Reply:
done